src/ducks/budgetAlerts/cli.js

```diff
@@ -71,7 +71,7 @@ main()
     // More information on open handles by decommenting the two following lines:
     // var wtf = require('wtfnode')
     // wtf.dump()
-    process.exit(0)
+    return process.exit(0)
```

Comment:
i am not sure this is working.

I think here the rule can be disabled. Or I suggest to:
```
process.exit(0)
return 0 // 0 as exit code
```